CoEpi/ui/onboarding/Constants.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Constants.swift
+//  CoEpi
+//
+//  Created by Robert Atherton on 26/03/2020.
+//  Copyright © 2020 org.coepi. All rights reserved.
+//
+
+import UIKit
+
+struct Constants {
+    
+    struct color {
+        
+        static let coepiBlue            = UIColor(named: "CoEpiBlue")!
```

Comment:
I would've suggested we stick with built in colors for now for simplicity, but given that this work is done, we might as well.